### DISCUSSION TITLE
Add a "Scan title" when importing a new scan

### DISCUSSION THREAD

**Discussion by Greenmouse94** [DefectDojo/django-DefectDojo#5003] [Ideas]
Hello,

It could be very helpful to add a title for each scan inside Findings > Import Scan Results.
And inside "Findings", for each finding, specified from which scan it comes from (filename or name).

I import a lot of semgrep scans and sometimes it is complicated to know from which file the findings come from, if the finding comes from the owasp-top-ten rulesets or the JavaScript rulesets etc.

Thank you.

**Comment by damiencarol** [comment]
Are you using the SARIF format or the semgrep one?

**Reply by Greenmouse94** [reply]
Hello,

The SEMGREP one.

**Comment by valentijnscholten** [comment]
@Greenmouse94  We added `test_title` support in https://github.com/DefectDojo/django-DefectDojo/pull/5342. Does that help?

**Reply by Greenmouse94** [reply]
Hello, 

Ok I'll try this.

Thank you.
